rdr_service/offline/biobank_samples_pipeline.py

```diff
@@ -86,7 +87,7 @@ def upsert_from_latest_csv():
         csv_reader = csv.DictReader(csv_file, delimiter="\t")
         written = _upsert_samples_from_csv(csv_reader)
 
-    ts = datetime.datetime.now()
+    ts = clock.CLOCK.now()
```

Comment:
Note: this is the method used to generate the timestamp applied to the `participant_summary` records in `upsert_from_latest_csv()`, so changed it from a datetime method here for consistency